verilog/formatting/tree_unwrapper.cc

```diff
@@ -1425,13 +1560,26 @@ void TreeUnwrapper::ReshapeTokenPartitions(
     case NodeEnum::kContinuousAssignmentStatement: {  // e.g. assign a=0, b=2;
       // TODO(fangism): group into above similar assignment statement cases?
       //   Cannot easily move now due to multiple-assignments.
+      bool anyAppendFittingSubpartitions = std::find_if(
```

Comment:
`const bool`

Reply:
Fixed. Btw. Is that partition policy propagation okay?

Reply:
It looks like "if any direct children nodes are append-fitting, then make this node also append-fitting (for these constructs)".

I _think_ that's ok.  It feels safe because these statements are isolated from one another.